extra-utils/autojump-rs/autobuild/beyond

```diff
@@ -0,0 +1,21 @@
+abinfo "Generating cheksum data for asset downloading ..."
+VER=22.5.3
+cat > release-$VER.sha256 << EOF
+00daf3698e17ac3ac788d529877c03ee80c3790472a85d0ed063ac3a354c37b1 release-v$VER.tar.gz
+EOF
+
+abinfo "Downloading assets ..."
+wget https://github.com/wting/autojump/archive/release-v$VER.tar.gz || \
+abdie "Failed fetching release-v$VER."
+sha256sum -c release-$VER.sha256 || abdie "Failed checksuming."
```

Comment:
Absolute path here.

Reply:
You could have not generated the checksum file here. See `extra-web/google-chrome` - I find that to be cleaner.